README.md

```diff
@@ -83,4 +83,20 @@ if __name__ == "__main__":
   opentracing.tracer.flush()
 ```
 
+### Thrift
+When using apache thrift rpc, make sure to both disable use_http by setting it to False as well
+as enabling use_thrift.
+
+```python
+return lightstep.Tracer(
```

Comment:
I still think that detailing the other options to this call makes it more difficult to consume the textual information. As a reader, I start wondering if the other parameters are relevant.

Reply:
Okay, can delete.